packages/syft/src/syft/serde/third_party.py

```diff
@@ -180,13 +179,42 @@ def serialize_bytes_io(io: BytesIO) -> bytes:
 except Exception:  # nosec
     pass
 
-# jax
-recursive_serde_register(
-    ArrayImpl,
-    serialize=lambda x: serialize(np.array(x), to_bytes=True),
-    deserialize=lambda x: jnp.array(deserialize(x, from_bytes=True)),
-)
 
+try:
```

Comment:
👍🏾

Reply:
Thank you very much for the kind words @shubham3121